hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/inventory/InventoryIdUtil.java

```diff
@@ -25,6 +25,16 @@
         public String feedId;
         public String managedServerName;
         public String idPart;
+
+        public String getFeedId() {
+            return feedId;
+        }
+        public String getManagedServerName() {
+            return managedServerName;
+        }
+        public String getIdPart() {
+            return idPart;
+        }
```

Comment:
if you want to make public getters, then make the data fields private so it forces people to go through the getters.